src/components/EthExchanges.js

```diff
@@ -549,6 +549,7 @@ const EthExchanges = () => {
         <Translation id="page-get-eth-exchanges-intro" />
       </Intro>
       <StyledSelect
+        aria-label={getDefaultMessage("page-get-eth-exchanges-search")}
```

Comment:
Using `getDefaultMessage` means that this will always resolve to English aria-labels. We should use `translateMessageId` here instead to make this accessible across languages. 

[See here for an example](https://github.com/ethereum/ethereum-org-website/blob/1829598d2a2e3819a7429bd9414db64b6b6724cc/src/components/Nav/Mobile.js#L269).

Reply:
Sure!